templates/theme-connect.php

```diff
@@ -0,0 +1,197 @@
+<?php
```

Comment:
@fajardoleo As mentioned in the original task description, there should only be one connect/opt-in template (unless there are too many differences). So templates/theme-connect.php is redundant and the relevant logic should be in `connect.php`.